wp-content/themes/ipbs-largo/less/single.less

```diff
@@ -10,4 +10,75 @@
             }
         }
     }
-}
\ No newline at end of file
+}
+// https://github.com/INN/umbrella-ipbs/issues/41
+.single-ipbs {
+	#page {
+		box-sizing: border-box; // make my life simpler
+	}
+	.entry-content {
+		box-sizing: border-box;
+		position: relative;
+
+		.alignleft {
+			box-sizing: border-box;
+		}
+		.alignwide {
+			box-sizing: border-box;
+		}
+		.alignfull {
+			margin-left: 0;
+			margin-right: 0;
+			transform: unset;
+			box-sizing: border-box;
+
+			width: 100vw;
+
+			margin-left: -2.5vw;
+			margin-right: -2.5vw;
+		}
+
+		@media ( min-width: 700px ) {
+			@width: ( 95vw );
+			@left: calc( ( @width - @measure ) * 5/12);
+			padding-left: calc( @left + @measure );
+			> * {
+			}
+			.alignleft {
+				width: @left;
+				float: left;
+			}
+			.alignwide {
+				width: calc( @left + @measure + 100% );
+				margin-right: 0;
+			}
+			.alignleft, .alignwide {
+				margin-left: calc( -1 * ( @left + @measure ) );
+			}
+			.alignfull {
+				margin-left: calc( -1 * ( @left + @measure + 2.5vw ) );
+			}
+		}
+		@media ( min-width: 1170px ) {
+			@width: 1170px;
+			@left: calc( ( @width - @measure ) * 5/12);
+			padding-left: calc( @left + @measure );
+			> * {
+			}
+			.alignleft {
+				width: @left;
+				float: left;
+			}
+			.alignwide {
+				width: calc( @left + @measure + 100% );
+				margin-right: 0;
+			}
+			.alignleft, .alignwide {
+				margin-left: calc( -1 * ( @left + @measure ) );
+			}
+			.alignfull {
+				margin-left: calc( -1 * (  @left + @measure + ( ( 100vw - 1170px )/2 )  ) );
+			}
```

Comment:
This is the present problem: the left margin is not as left as it should be.

Reply:
If you change `@left` to `@width + @measure` for this specific instance, it should fix it.

But, this Largo style will need to be updated
```
@media screen and (min-width: 1600px)
#page {
   padding: 0 20px;
}
```
to start at 1200px instead of 1600px.

![Screen Shot 2020-02-04 at 12 17 08 PM](https://user-images.githubusercontent.com/18353636/73769218-48092e00-4748-11ea-9c51-f30f697c7a17.png)

Reply:
Actually, just removing that `#page` padding altogether works too.

Reply:
The changing padding on `#page` was the clue here. I also found a similar problem between 769px and 979px.

Fixed in 0a3ca5a